### DISCUSSION TITLE
Which driver should I download?

### DISCUSSION THREAD

**Discussion by RKI91** [pal1000/Realtek-UAD-generic#133] [Q&A]
I'm a bit confused by the lastest versions (6.0.9472.1 and 6.0.9475.1) Which one should I choose for HD Audio Device? From the description I understand that one of the versions is for Intel Audio Devices and one is for HD Audio Devices but I don't know which one is which. The version numbers in the description do not match the download version numbers. Please help.

**Comment by pal1000** [comment]
6.0.9475.1 is the latest. I sometimes release in batches so 6.0.9472.1 and 6.0.9475.1 went out together. They both support HDAUDIO and INTELAUDIO devices.
